tools/super-tox.py

```diff
@@ -4,27 +4,57 @@
 
 import asyncio
 import contextlib
+import dataclasses
+import itertools
 import logging
 import os
 import pathlib
+import pprint
 import re
 import shutil
 import subprocess
 import sys
-import tomllib
+from contextlib import nullcontext
+
+try:
+    import tomllib  # type: ignore
+except ImportError:
+    import tomli as tomllib  # type: ignore
```

Comment:
Is tomli significantly better? If so I'd rather just put it in the requirements and always use it.

Reply:
Basically tomli was added to stdlib under tomllib name in Py 3.11.

This project supports wider range Python bases, so prefer native, fall back to external dep if run under older Python.

Reply:
I don't think super-tox itself needs to run under older Pythons (I would be fine with >=3.12). The tests do, but they'll get their Python version from where `tox` is installed unless the config specifies something.

What I would like (#3) is to be able to specify which Python is used, but the super-tox script itself would still work with something modern.

But I'm fine leaving this as-is for the PR.